README.md

```diff
@@ -266,8 +267,8 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [fastlane](https://fastlane.tools)
 - [Feeds](http://www.feedsapp.com/)
 - [FileZilla](https://filezilla-project.org/)
-- [Fish](http://ridiculousfish.com/shell/)
 - [Fisherman](http://fisherman.sh/)
+- [Fish](http://ridiculousfish.com/shell/)
```

Comment:
Up there, you put `Bash` before `Bash it`, but here you put `Fish` after `Fisherman`. Let's be consistent ;)